_posts/2020-07-15-june-rubygems-update.md

```diff
@@ -0,0 +1,53 @@
+---
+title: June 2020 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in June.
+
+
+## RubyGems News
+
+In June, RubyGems saw significant improvements on Bundler and RubyGems development. We worked on synchronizing the latest versions of both libraries with ruby-core, and fixed issues that came up. We also [enforced our ruby-core workflow](https://github.com/rubygems/rubygems/pull/3725) to try to prevent further tedious integrations with ruby-core in the future by catching issues early and minimizing the back and forth of patches between upstream and downstream.
```

Comment:
I would mention initially, since it's the most important thing for users, that we released rubygems 3.2.0.rc.1 and bundler 2.2.0.rc.1. And then on a separate paragraph add all the detailed tasks that were done.

Reply:
Sure! I think that's a better structure.